src/qt/overviewpage.cpp

```diff
@@ -620,15 +614,9 @@ void OverviewPage::togglePrivateSend(){
         settings.setValue("hasMixed", "hasMixed");
     }
 
-    auto it = privateSendClientManagers.find(walletModel->getWalletName().toStdString());
-    if (it == privateSendClientManagers.end()) {
-        // nothing to do
-        return;
-    }
-    CPrivateSendClientManager* privateSendClientManager = it->second;
-
-    if (!privateSendClientManager->IsMixing()) {
-        const CAmount nMinAmount = CPrivateSend::GetSmallestDenomination() + CPrivateSend::GetMaxCollateralAmount();
+    if (!walletModel->privateSend().isMixing()) {
+        auto& options = walletModel->node().privateSendOptions();
+        const CAmount nMinAmount = options.getSmallestDenomination() + options.getMaxCollateralAmount();
```

Comment:
Why max collateral amount? @UdjinM6 (I see you're maintaining old logic, so this isn't really a problem for you @xdustinface)